src/bin/cargo-flamegraph.rs

```diff
@@ -113,10 +114,18 @@ enum Opts {
 
 fn build(opt: &Opt) {
     let mut cmd = std::process::Command::new("cargo");
-    cmd.arg("build");
 
-    if !opt.dev {
-        cmd.arg("--release");
+    if let Some(ref bench) = opt.bench {
+        cmd.arg("bench");
+        cmd.arg("--no-run");
+        cmd.arg("--bench");
+        cmd.arg(bench);
```

Comment:
I find `bench --no-run` pretty unintuitive. It seems like we could do `build --profile bench` instead?

Reply:
Could you please check it on your setup? I have the following error.

```
❯ cargo build --profile bench
error: Usage of `--profile` requires `-Z unstable-options`
```

Reply:
```
❯ cargo --version
cargo 1.50.0 (f04e7fab7 2021-02-04)
```

Reply:
Yeah, you're right -- weird that it shows it in the `--help` even when it's not actually available.

Reply:
Okay, in that case can you just add a comment here to make it clearer what this is trying to do (namely, building with the `bench` profile)?

Reply:
Yes, sure.